packages/wdio-cli/src/launcher.ts

```diff
@@ -56,6 +56,9 @@ class Launcher {
         private _isWatchMode = false
     ) {
         this.configParser = new ConfigParser()
+        if ( this.configParser && this.configParser.autoCompile ) {
```

Comment:
Would that ever be undefined?
```suggestion
        if (this.configParser.autoCompile) {
```